dom.bs

```diff
@@ -642,6 +646,9 @@ attributes must return the values they were initialized to. When an
 <a>event</a> is created the attributes must be
 initialized to null.
 
+The <dfn attribute for=Event><code>srcElement</code></dfn>
+attribute must return the {{Event/target}} attribute value.
```

Comment:
I'd prefer to define it together with target. So "The target and srcElement attributes must return X".